src/main.ts

```diff
@@ -87,6 +88,91 @@ function makeExtendViewer() {
   }
 }
 
+function observeSegmentSelect(targetNode: Element) {
+  // Select the node that will be observed for mutations
+  if (!targetNode) {
+    return;
+  }
+
+  // Options for the observer (which mutations to observe)
+  const config = {childList: true, subtree: true};
+
+  const createChangelogButton = (segmentIDString:
+                                     string): HTMLButtonElement => {
+    // Button for the user to copy a segment's ID
+    const changelogButton = document.createElement('button');
+    changelogButton.className = 'nge-segment-changelog-button';
+    changelogButton.title = `Show changelog for Segment: ${segmentIDString}`;
+    changelogButton.innerHTML = '💡';
+    changelogButton.addEventListener('click', async () => {
+      // changelogButton.disabled = true;
+
+      const request =
+          `https://prodv1.flywire-daf.com/segmentation/api/v1/table/fly_v31/root/${
```

Comment:
It's not always going to be `fly_v31`, this needs to be dependent on the dataset